src/pages/_document.tsx

```diff
@@ -19,7 +19,7 @@ function MyDocument() {
         />
         <meta property='og:image' content='/assets/images/logo-mocha.png' />
         <meta property='og:locale' content='en_US' key='meta-locale' />
-        <meta property='og:url' content='https://mocha.chroma-gaming.xyz/' />
+        <meta property='og:url' content='https://mocha-bot.xyz/' />
```

Comment:
put in env would be better

Reply:
not in priority, would be moved in the next PR